introduction.md

```diff
@@ -1 +1,4 @@
 Hello, my name is Aaron. My favorite CodeSchool path is the JavaScript path.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?